site/config.yaml

```
@@ -140,9 +142,12 @@ languages:
         - name: Help-Center
           url: https://help.skribble.com/english
           weight: 10
-        - name: QES explained
-          url: /qes-explained/
+        - name: Signaturstandards
```

```suggestion
        - name: Signature standards
```
missing E in signature and split into two words

---

I corrected it. Thanks!